generators/voxel_generator_test.cpp

```diff
@@ -101,7 +115,7 @@ void VoxelGeneratorTest::generate_block_waves(VoxelBuffer &out_buffer, Vector3i
 					rh = size.y;
 
 				for (int ry = 0; ry < rh; ++ry) {
-					out_buffer.set_voxel(_voxel_type, rx, ry, rz, 0);
+					out_buffer.set_voxel(_voxel_type, rx, ry, rz, _channel);
```

Comment:
For SDF this is not correct

Reply:
eh, this was just to get it functional. It currently only supports blocky and doesn't work at all with VT/smooth, or VLT. The whole thing needs work. It doesn't handle LODs on VLT and looks janky. My aim with this PR was not to get it perfect but to get it functional and draw something in every situation.

Reply:
If so, remember to open an issue so it can be tracked

Reply:
will do